### ISSUE TITLE
Make a demo React App using this module to show GitHub Star's feed

### ISSUE THREAD

**Issue by vinitshahdeo** [vinitshahdeo/github-stars-feed#12]
```
npm i github-stars-feed
```

As `github-stars-feed` is available as an NPM module, can we make a demo React App to show GitHub Star's feed using card view.

- Refer [Cards](https://react-bootstrap.github.io/components/cards/)
- Check `github-stars-feed` on [NPM](https://www.npmjs.com/package/github-stars-feed)
- [Learn more](https://vinitshahdeo.dev/github-stars-feed)

You can [Create React App](https://github.com/facebook/create-react-app) to set up the React app by running one command.

```bash
npx create-react-app github-stars-feed
cd github-stars-feed
npm start
```

**Comment by tanmayVaish** [comment]
May I get assigned on this issue. I can work on this issue!

**Comment by parthn2** [comment]
@vinitshahdeo I would like to work on this!

**Comment by koushikpuppala** [comment]
@vinitshahdeo I would like to work on this! I have made a few react apps also. So I think I can work on this. You can check my portfolio [https://koushikpuppala.live](https://koushikpuppala.live)

**Comment by ayush-sys** [comment]
I would like to work on this issue. I have good experience in React & JavaScript.

**Comment by ayasahmad63** [comment]
can you assign this to me?

**Comment by Sagargajare** [comment]
Hi, I know reactjs. and I want to contribute to this project.

**Comment by tanmayVaish** [comment]
@vinitshahdeo I have created demo react app in Pull Request #13 . Kindly review it!!!

**Comment by vinitshahdeo** [comment]
Hey folks, please feel free to raise your PRs. I will try to merge them all. Just ensure that you're not making the same thing - every demo app should be different in terms of look & feel.

**Comment by tanmayVaish** [comment]
> Hey folks, please feel free to raise your PRs. I will try to merge them all. Just ensure that you're not making the same thing - every demo app should be different in terms of look & feel.

I think you can link only 1 issue with 1 PR, I am confused, 

How are you going to make multiple merges to your repo?
How are you suppose to close this one if you are accepting multiple PR?

**Comment by vinitshahdeo** [comment]
@tanmayVaish You're right. But we can have multiple demo apps that can be different in terms of UI/UX design. The aim is not just to complete the Hacktoberfest challenge.

**Comment by omkarvw** [comment]
Hello Sir, I would like to try contributing to this app. I am new to open source and would like to learn this and contribute to it.
So can I try contributing during hacktoberfest2022 ?
